omnigibson/tasks/point_navigation_task.py

```diff
@@ -68,7 +68,7 @@ def __init__(
         floor=0,
         initial_pos=None,
         initial_quat=None,
-        goal_pos=None,
+        goal_pos=np.full(3, 0.0),
```

Comment:
is there a reason to use this vs. np.zeros(3)?

Reply:
it should in fact default to None with the default value created in init so that the same goal pos array is not shared between different instances.